hysds/user_rules_dataset.py

```diff
@@ -89,6 +89,13 @@ def update_query(_id, system_version, rule):
     return final_query
 
 
+@backoff.on_exception(
+    backoff.expo, Exception, max_tries=backoff_max_tries, max_value=backoff_max_value
+)
+def search_es(index, body):
+    return grq_es.es.search(index=index, body=body)
```

Comment:
i'd recommend adding a `timeout` here as well
https://elasticsearch-py.readthedocs.io/en/v7.14.0/api.html#elasticsearch.Elasticsearch.search
> timeout – Explicit operation timeout

Reply:
@DustinKLo Should we set this timeout=30?

Reply:
30 sounds good

Reply:
@mkarim2017 what do you think?

Reply:
@mcayanan my mistake, I was looking at another ES call, not search. Updated the timeout to 30 sec.